doc/mgmt/Management Framework.md

```diff
@@ -2552,7 +2552,7 @@ Following are the list of Open source tools used in Management framework
 ## 12 Appendix B
 
 Following are the list of Open source libraries used in telemetry container. 
-Always refer to the [Makefile](https://github.com/Azure/sonic-telemetry/blob/master/Makefile) for the sonic-telemetry container for current package list.
```

Comment:
>Azure/sonic-telemetry [](http://example.com/codeflow?start=50&length=21)

This repo is migrated in different way:
Azure/sonic-telemetry -> sonic-net/sonic-gnmi

There are 4 occurrences.

Reply:
Updated.